pages/release-notes-archive.md

```diff
@@ -1,10 +1,10 @@
 ---
 layout: page
-title: Fluo Release Notes Archive
+title: Release Notes Archive
 permalink: "/release-notes/"
 ---
 
-Below are the release notes for all Fluo releases:
+Below are the release notes for all Apache Fluo releases:
```

Comment:
In addition to making things clear on the pages that link to info about old releases... we should probably also add something to the pages that actually have the content